README.md

```diff
@@ -79,6 +79,7 @@ Variable | Description | Examples/Values | Default
  `DEPLOYMENT_CIRCUIT_BREAKER_RULE` | Enable or disable circuit breaker | `enable=true,rollback=true`
  `ECS_CONTAINER_STOP_TIMEOUT` | Set stopTimeout on taskdefinition | min: 0, max: 120, default: 30
  `TZ`| Set this variable to the desired task timezone | America/Sao_Paulo
+ `WORKLOAD_PATH`| Set the workload path to run build commands on correct directory| ./Projects/my-app, ./
```

Comment:
"Set the workload path to run build commands on ~~correct directory~~" should do it

Reply:
Changed to new description.